src/main/java/com/w6/nlp/Parser.java

```diff
@@ -53,6 +53,9 @@ else if (parent.label().value().startsWith("N"))
         where = LocationParser.parseLocationFromString(input);
         
         when = DateTimeParser.parseDateAndTimeFromString(input);
+        
+        ViolentVerbsParser violentVerbsParser = new ViolentVerbsParser();
```

Comment:
Create in in a constructor of Parser, we don't to create new instance for each request.